src/UnityExtension/Assets/Editor/GitHub.Unity/EntryPoint.cs

```diff
@@ -53,7 +53,7 @@ private static void Initialize()
                     Logging.Error(ex, "Error rotating log files");
                 }
 
-                Debug.Log("Initializing GitHub for Unity log file: " + logPath);
```

Comment:
Oooh, this is an excellent chance to combine this message with the one above so we don't log into the console twice, there was someone complaining about that. How about

`Debug.LogFormat("Initialized GitHub for Unity version {0}. Log file: {1}", ApplicationInfo.Version, logPath);`?